config/src/config/test_data/public_full_node.yaml

```diff
@@ -12,6 +12,7 @@ full_node_networks:
     - discovery_method: "onchain"
       listen_address: "/ip4/0.0.0.0/tcp/6180"
       network_id: "public"
+      max_frame_size: 8388608 # 8 MiB
```

Comment:
why use MiB and not MB btw :P ?

Reply:
The original limit in the code is 8 MiB, so I didn't want to change it.

Reply:
8 MiB == 8388608 bytes
8 MB == 8000000 bytes

: p

Reply:
yeah, that's why 8 MB is better imo :P